provider/app.py

```diff
@@ -65,6 +66,13 @@ def testRoute():
     return jsonify('pong')
 
 
+@app.route('/health')
+def healthRoute():
+    currentTime = datetime.now()
+    diff = currentTime - startTime
+    return jsonify({'uptime': diff.seconds})
```

Comment:
I think at some point we could add more information to the JSON that is returned here. Possibilities include: how many triggers have been invoked, how many triggers failed to be invoked, how many consumers are there, etc.